src/AM.Condo/Targets/Publish/Unity.targets

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="UnityPublish" Condition=" $(UnityPublish) ">
+    <Exec Command="&quot;$(UnityPath)&quot; -quit -batchmode -nographics -projectPath &quot;@(UnityProjectPath)&quot; -buildTarget $(BUILD_TARGET) -customBuildTarget $(BUILD_TARGET) -customBuildPath &quot;$(PublishArtifactsRoot)$(BUILD_TARGET.ToLower())&quot; -customBuildOptions AcceptExternalModificationsToPlayer -executeMethod Build.PerformBuild -logFile -noUpm"
```

Comment:
Is the nographics switch common? Should it be an option?

Reply:
Same for custom build target?

Reply:
Same for additional flags, like options... should these be defaulted but still overridable ala DOTNET_TEST_OPTIONS, etc?

Reply:
`-quit -batchmode -nographics` are all defined in the unity docs for running a CI build. If they are not set then unity will not close the process when complete, and open a GUI interface

Still playing around with customBuildTarget. to run unity headless you need to have an entry point file. 

adding the additional flags is a good idea

Reply:
docs: https://docs.unity3d.com/Manual/CommandLineArguments.html